luminaire/model/lad_structural.py

```diff
@@ -442,12 +436,72 @@ def _training(self, data, ts_start, ts_end, min_ts_length=None, min_ts_mean=None
 
         return result, order
 
-    def train(self, data, optimize=False, **kwargs):
+    def _validate_model(self, data, hyper_params, result):
+        """
+        This function validates a newly trained model before publishing
+
+        :param pandas.DataFrame data: Input time series data
+        :param dict hyper_params: Hyperparameters for model training
+        :param dict result: Training outputs
+        :return: A validation flag for model to publish
```

Comment:
Nit: So a Yes means good to publish, might be good to explicitly add that details too in the commet.

Reply:
Documentation added